andela/travel_tool_back
# #160843273 Fix view-details link on notifications for posted comments

#### What does this PR do?
This PR fixes the link for ```View Details``` on notifications for comments posted on requests.
 
#### Description of Task to be completed?
- change ```View Details``` link on notifications for posted comments on requests

#### How should this be manually tested?
Steps to reproduce and test the fix
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
 - Enter into the project directory - `cd travel_tool_back`
 - Start the server - `make start`
-  Set up the clientside using the guide on ```https://github.com/andela/travel_tool_front```
 - Navigate to [travela](http://travela-local.andela.com:3000/#) in your browser
- Login into the application
- Comment on a request you are a manager of.
- Check to see the notification created for it in the notification pane

The link ```View details``` should take you to the right request URL(```../requests/{id}```). However, it takes you to  ```../my-requests{id}``` which is a dead link.


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160843273](https://www.pivotaltracker.com/story/show/160843273)

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
